subprojects/android-test/test-report/src/main/kotlin/com/avito/android/test/report/synchronoius_mage_uploader/SynchronoiusImageUploader.kt

```diff
@@ -0,0 +1,44 @@
+import android.annotation.SuppressLint
+import com.avito.filestorage.FutureValue
+import com.avito.filestorage.RemoteStorage
+import com.avito.logger.Logger
+import java.io.File
+
+@SuppressLint("LogNotTimber")
+interface SynchronoiusImageUploader {
```

Comment:
I propose to name it `ImageUploader`. 
Asynchronicity is already expressed by a return type `upload(referenceImage: File): String <--`

Reply:
Remove